### DISCUSSION TITLE
Animated Bubble Disperse Button

### DISCUSSION THREAD

**Discussion by richieracoon** [sButtons/sbuttons#1292] [Ideas]
Hey,
I suggested a similar idea. I'm sorry I couldn't work on it at the time but I'd really like to start over please. 
Could you reopen that issue for me?
https://github.com/sButtons/sbuttons/issues/1147
or if it's possible I'd rather like to work on this one as it seems easiert to grasp:
https://github.com/sButtons/sbuttons/issues/225

Kind regards

**Comment by shahednasser** [comment]
Hello @richieracoon I have reopened the issue you were assigned to. Let me know if you want to work on the other issue mentioned about the like button (although the issue description might change a little now)

**Reply by richieracoon** [reply]
Hello Shahed,
as mentioned above I'd rather work on the  Like-Buttons because I think it'll  be a little easier for me and if I can still afford to spend the time I'd like to do the Issue #1147 after that. How would the description of  #225 change now?

**Reply by shahednasser** [reply]
@richieracoon if you check sButtons now we have changed all icon buttons into one basic class which is the icon-btn class. You either have to adapt to that class, or do something similar to the star button

**Reply by richieracoon** [reply]
@shahednasser I have programmed a possible solution for #225 and made a pull request for it.
